Core/Services/StitchCoreRemoteMongoDBService/Tests/StitchCoreRemoteMongoDBServiceTests/Sync/ReadWriteLockTests.swift

```diff
@@ -1,190 +1,262 @@
-import Foundation
+/*******************************************************************************
```

Comment:
[nit] put in a "Vendor" or "AtomicKit" subdir

Reply:
I'll make an AtomicKit sub-dir. I've altered the files to some extent, so they aren't purely vendored.